Jenkinsfile

```diff
@@ -66,20 +67,21 @@ pipeline {
     }
     stage('Run tests') {
       steps {
+        setGitHubBuildStatus('test', 'Unit tests', 'PENDING')
         container('nodejs') {
           echo """
           ---------
           Run tests
           ---------"""
-          sh 'npm run test -- --debugBrowsers=ChromeHeadlessNoSandbox'
+          sh 'SAUCE_USERNAME=nuxeo-elements SAUCE_ACCESS_KEY="$(jx step credential -s saucelabs-elements -k key)" npm run test'
```

Comment:
Wouldn't it be clearer to use `withEnv` and/or `withCredentials`?

Reply:
done 👍